pom.xml

```diff
@@ -42,6 +42,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <jenkins.version>1.651.2</jenkins.version>
```

Comment:
For versions between 1.651 and 2.3 I'd say.

I wonder if I should not only rely on reflection to call `getAllParameters` and drop the core dependency bump.